build.gradle

```diff
@@ -18,11 +17,12 @@ buildscript {
       'kotlin': '1.3.31',
       'moshi': '1.8.0',
       'okio': '2.2.2',
-      'ktlint': '0.31.0'
+      'ktlint': '0.31.0',
+      'picocli': '4.0.0-beta-1b'
```

Comment:
Should we wait for a proper release?

Reply:
Sounds good, I'll put this on a branch or similar until then.  Test with that.